Modelica/Blocks/Noise.mo

```diff
@@ -588,7 +588,7 @@ the desired situation. For this purpose the following parameters can be defined:
          of the instances are the same).<br>
          If <strong>useAutomaticLocalSeed = true</strong>, the
          local seed is determined automatically using a hash value of the instance name of the model that is
-         inquired with the Modelica built-in operator <a href=\"https://specification.modelica.org/v3.4/Ch3.html#getinstancename\">getInstanceName()</a>.
+         inquired with the Modelica built-in operator <a href=\"https://specification.modelica.org/maint/3.6/operators-and-expressions.html#modelica:getInstanceName\">getInstanceName()</a>.
```

Comment:
Because the document is poorly structured.  Now that [the section](https://specification.modelica.org/maint/3.6/operators-and-expressions.html#derivative-and-special-purpose-operators-with-function-syntax) starts with an overview table of the operators described in the section, it would have made more sense if at least the fairly short sections on `delay` and `getInstanceName` were included directly under the respective main entry points linked from the table.  This would make 3.7.4 more similarly structured to all the other sections under 3.7.

Until then, and in the case of `getInstanceName`, I agree that there is not a whole lot to see at the currently linked main entry point.  I'll change it the way you suggest.

Reply:
Done.  Resolving.